engine/execution/computation/computer/result_collector.go

```diff
@@ -274,6 +279,14 @@ func (collector *resultCollector) processTransactionResult(
 		return nil
 	}
 
+	// TODO(ramtin): trigger consumers concurrently
+	for _, consumer := range collector.consumers {
```

Comment:
move this loop into commitCollection()

Reply:
Ah, nice, I wrote this before commitCollection was here :D, moving it to the commitCollection

Reply:
updated at [066683c](https://github.com/onflow/flow-go/pull/4049/commits/066683c63c4de690855a8e6443d9a51c769e12dc)